picard/file.py

```diff
@@ -627,36 +627,36 @@ def _apply_additional_files_moves(self, moves):
                           new_file_path, why)
 
     def remove(self, from_parent=True):
-        if from_parent and self.parent:
-            log.debug("Removing %r from %r", self, self.parent)
-            self.parent.remove_file(self)
+        if from_parent and self.parent_item:
+            log.debug("Removing %r from %r", self, self.parent_item)
+            self.parent_item.remove_file(self)
         self.tagger.acoustidmanager.remove(self)
         self.state = File.REMOVED
 
     def move(self, parent):
         # To be able to move a file the target must implement add_file(file)
-        if hasattr(parent, 'add_file') and parent != self.parent:
-            log.debug("Moving %r from %r to %r", self, self.parent, parent)
+        if hasattr(parent, 'add_file') and parent != self.parent_item:
```

Comment:
For consistency, we should also rename `parent` argument, I suggest indicating direction too for all those methods: `to_parent_item` , `from_parent_item`

Reply:
See 2369431e4713d6c1acff44e6342cfe992daabaf7